.travis.yml

```diff
@@ -8,26 +8,33 @@ php:
   - 7.0
   - hhvm
 
-matrix:
-  allow_failures:
-    - php: 7.0
-    - php: hhvm
+sudo: false
 
 cache:
   directories:
     - $HOME/.composer/cache
+    - vendor
+
+matrix:
+  include:
+    - php: 5.4
+      env: COMPOSER_FLAGS="--prefer-lowest"
+  allow_failures:
+    - php: 7.0
+    - php: hhvm
 
 before_script:
-  - composer self-update
-  - composer install --prefer-source --no-interaction
+  - mkdir -p ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d && echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+  - composer selfupdate
```

Comment:
Re-reading Travis-CI documentation, you may found that

> (...) we update composer every time we update the PHP build environment, which is every 30-60 days. Because composer has a time-based update warning, you may see messages (related to `30 days old` composer build), which may be safely ignored.

We may omit that step so far.